frontend/webpack.common.js

```diff
@@ -14,6 +14,9 @@ module.exports = {
   resolveLoader: {
     fallback: path.join(__dirname, 'node_modules'),
   },
+  devServer: {
```

Comment:
What does it do? I didn't have problems accessing by now, but I didn't test access much